internal/util/util.go

```diff
@@ -16,12 +16,32 @@ const (
 	CephUnknown
 )
 
+// List of known CephVersion strings
+const (
+	Nautilus = "nautilus"
+	Octopus  = "octopus"
+	Pacific  = "pacific"
+	Quincy   = "quincy"
+	Reef     = "reef"
+	Squid    = "squid"
+	Main     = "main"
```

Comment:
Might as well add "tentacle" here because it's going to be created any day now. It won't be immediately used but it will be harmless. And since we know what the name's going to be it will avoid having to do another PR in a week or two.